code/src/Components/AlbumCover.js

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from 'react'
+import Radium from 'radium'
+import styles from '../styles'
+import AlbumCoverBackground from './AlbumCoverBackground'
+import AlbumCoverIcons from './AlbumCoverIcons'
+
+const AlbumCover = (props) => {
+  const [hoverBackgroundVisible, setHoverBackgroundVisible] = useState(false)
+  const [underline, setUnderline] = useState(false)
+  return (
+    <div>
+      <div style={styles.imgWrapper} onMouseEnter={() => setHoverBackgroundVisible(true)} onMouseLeave={() => setHoverBackgroundVisible(false)}>
+        <a href={props.linkAlbum}>
+          <img style={styles.img} src={props.image} alt="Album cover"></img>
+          {hoverBackgroundVisible && <AlbumCoverBackground />}
+          {hoverBackgroundVisible && <AlbumCoverIcons />}
+        </a>
+      </div>
+
+      <a href={props.linkAlbum} style={styles.linkText}>
+        <h2 key={"Album"} style={styles.h2}>{props.name}</h2>
+      </a>
+      <a href={props.linkArtist} style={styles.linkText}>
+        <h3 key={"Artist"} style={styles.h3}>{props.artist}</h3>
+      </a>
+    </div>
+  )
+}
+export default Radium(AlbumCover)
```

Comment:
What does this Radium thing do?

Reply:
Radium enables you to write CSS in JS. My mentor explained that they used this maybe three years ago and that it would be beneficial for me to learn as a middle step before learning how they style today.